spec/Section 3 -- Type System.md

```diff
@@ -232,6 +232,152 @@ Schema extensions have the potential to be invalid if incorrectly defined.
 2. Any non-repeatable directives provided must not already apply to the
    original Schema.
 
+### Schema Coordinates
+
+Schema Coordinates are human readable strings that uniquely identify an element defined in a GraphQL Schema.
+
+**Definition**
+
+SchemaCoordinate:
+  - Name
+  - Name . Name
+  - Name . Name ( Name : )
+  - @ Name
+  - @ Name ( Name : )
+
+**Semantics**
+
+SchemaCoordinate: Name
+  1. Let {typeName} be the value of the first {Name}.
+  2. Return the type in the schema named {typeName}.
```

Comment:
Assuming that "type in the schema" applies to scalar types too? Is this worth clarifying?

Reply:
I don't think this needs clarification.